docs/source/examples/LeedsSportsPose.rst

```diff
@@ -0,0 +1,183 @@
+
+Note that after creating the :ref:`reference/dataset_structure:Dataset`,
+you need to load the :ref:`reference/dataset_structure:catalog`.(L42)
+The catalog file "catalog.json" is in the same directory with dataloader file.
+
+In this example, we create a default segment without giving a specific name.
+You can also create a segment by ``dataset.create_segment(SEGMENT_NAME)``.
+
+See :ref:`this page <reference/label_format:Keypoints2D>` for more details for about Keypoints2D annotation details.
+
+.. note::
+   The :ref:`LeedsSportsPose dataloader <LeedsSportsPose-dataloader>` above uses relative import(L11-13).
+   However, when you write your own dataloader you should use regular import.
+   And when you want to contribute your own dataloader, remember to use relative import.
+
+****************
+ Upload Dataset
+****************
+
+After you finish the :ref:`reference/glossary:Dataloader` and organize the "LeedsSportsPose" into a
+:class:`~tensorbay.dataset.dataset.Dataset` object, you can upload it
+to TensorBay for sharing, reuse, etc.
+
+.. code:: python
+
+   dataset_client = gas.upload_dataset(dataset, jobs=8, skip_uploaded_files=False)
+   dataset_client.commit("LeedsSportsPose")
+
+Remember to execute the :ref:`features/version_control:Commit` step after uploading.
+If needed, you can re-upload and commit again.
+Please see :ref:`this page <features/version_control:Version Control>` for more details about version control.
+
+.. note::
+
+   Commit operation can also be done on our GAS_ Platform.
+
+.. _gas: https://www.graviti.cn/tensorBay
+
+**************
+ Read Dataset
+**************
+
+Now you can read "LeedsSportsPose" dataset from TensorBay.
+
+.. code:: python
+
+   dataset_client = gas.get_dataset("LeedsSportsPose")
+
+In :ref:`reference/dataset_structure:Dataset` "LeedsSportsPose", there is one default
+:ref:`Segments <reference/dataset_structure:Segment>` ``""`` (empty string). You can get it by passing the segment name.
+
+.. code:: python
+
+   from tensorbay.dataset import Segment
+   default_segment = Segment("", dataset_client)
+
+In the train :ref:`reference/dataset_structure:Segment`, there is a sequence of :ref:`reference/dataset_structure:Data`. You
+can get one by index.
+
+.. code:: python
+
+   data = default_segment[0]
+
+.. note::
+
+   If the :ref:`reference/dataset_structure:Segment` or
+   :ref:`advanced_features/fusion_dataset/fusion_dataset_structure:fusion segment`
+   is created  without given name, then its name will be "".
+
+In each :ref:`reference/dataset_structure:Data`,
+there is a sequence of :ref:`reference/label_format:Keypoints2D` annotations.
+You can get one by index.
+
+.. code:: python
+
+   label_keypoints2d = data.label.keypoints2d[0]
+   x = data.label.keypoints2d[0][0].x
+   y = data.label.keypoints2d[0][0].y
+   z = data.label.keypoints2d[0][0].z
+
+There is only one label type in "LeedsSportsPose" dataset, which is ``keypoints2d``. The information stored in ``x`` is
```

Comment:
The information stored in ``x``(``y``) is
the x (y) coordinate of one keypoint of one keypoints list.